packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -45,23 +48,47 @@ const propTypes = {
    */
   columnHeaderHeight: PropTypes.string,
   /**
-   * String that specifies the default height for rows in the grid. Any valid CSS width value is accepted.
-   * To override the default value, provide height for the row that needs to be overridden.
+   * String that specifies the height for the rows in the grid. Any valid CSS value is accepted.
    */
   rowHeight: PropTypes.string,
   /**
    * Number indicating the index of the column that represents row header. Index is 0 based and cannot exceed one less than the number of columns in the grid.
    */
   rowHeaderIndex: PropTypes.number,
+  /**
+   * Callback function that is called when a selectable cell is selected. Parameters: `onRowSelect(rowId, columnId)`.
+   */
+  onCellSelect: PropTypes.func,
+  /**
+   * Function that is called when a selectable header cell is selected. Parameters: `onColumnSelect(columnId)`
+   */
+  onRowSelect: PropTypes.func,
+  /**
+   * Callback function that will be called when all rows are selected. Parameters: `onSelectAllRows()`.
+   */
+  onRowSelectAll: PropTypes.func,
   /**
    * Function that is called when a selectable header cell is selected. Parameters: `onColumnSelect(columnId)`
    */
   onColumnSelect: PropTypes.func,
+  /**
+   * Callback function that is called when all selected rows need to be unselected.
+   */
+  onClearSelectedRows: PropTypes.func,
+  /**
+   * Callback function that is called when no row is selected and the row selection mode needs to turned off.
+   */
+  onDisableSelectableRows: PropTypes.func,
```

Comment:
Should these prop description also be providing the params?

Reply:
Updated